### PR TITLE
moving docs into the new location the registry prefers

### PR DESCRIPTION
**Pull request by attachmentgenie**
the registry docs suggest the docs location needs to change, this PR provides the proposed location change

related to #231

### CODE REVIEW

**Comment by emesika** [comment]
can you please elaborate on this change , I don't understand what is changed here , please specify what was the problem that you want to solve and where this information was located before that change

**Comment by attachmentgenie** [comment]
@emesika the changes i made are based on the following [guidance](https://www.terraform.io/docs/registry/providers/docs.html#migrating-legacy-providers-docs) from the HashiCorp provider documentation. With the release of the terraform provider registry they deprecated the documentation location in favor of the locations in this PR

File: docs/resources/vnic.html.md [hunk 1]
```diff
@@ -1,11 +1,3 @@
----
```
Comment (emesika) [inline docs/resources/vnic.html.md:1]: Shouldn't file name be just vnic.md  ???

Comment (attachmentgenie) [reply docs/resources/vnic.html.md]: you are absolutely right, i pushed a new commit fixing this

**Pull request merged**